crates/holochain_deepkey_dna/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "holochain_deepkey_dna"
+version = "0.0.1"
+edition = "2021"
+authors = ["Michael dougherty <michael.dougherty@holo.host>"]
+license = "CAL-1.0"
+repository = "https://github.com/holochain/deepkey"
+description = "A compilation of the Deepkey DNA for use in Holochain"
+
+include = [
+    "*",
+    "../../dnas/deepkey.dna"
```

Comment:
If we are copying the dna file into the `src` directory, do we still need this reference to the parent location?